.github/workflows/ci-checklist.yml

```
@@ -0,0 +1,38 @@
+﻿name: ci-shotcaller-checklist
+
+on:
+  workflow_call:
+    inputs:
+      project-path:
+        type: string
+        required: true
+        description: Path to the csproj file
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)